go/codeql-tools/configure-baseline.cmd

```diff
@@ -1,6 +1,7 @@
 @echo off
-if exist vendor\modules.txt (
-  type "%CODEQL_EXTRACTOR_GO_ROOT%\tools\baseline-config-vendor.json"
-) else (
-  type "%CODEQL_EXTRACTOR_GO_ROOT%\tools\baseline-config-empty.json"
-)
+SETLOCAL EnableDelayedExpansion
```

Comment:
What does this do? Copilot told me something about expanding variables with ! instead of %, but I don't see that being used.

Reply:
A quick search for this suggests that it is more about when variables are expanded in "lines" of the script. Apparently, by default, variables are expanded before each "line" is executed. That is a problem where a "line" contains e.g. a loop and a variable has to be expanded for each iteration. 

That said, I also don't really see a need for it in this script.